Dockerfile

```diff
@@ -145,8 +144,8 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         php7.4-zip \
         php7.4-intl \
         pngcrush \
-        python-setuptools \
         python3.8-dev \
```

Comment:
you might wanna remove 3.8, no need to have both that and 3.9?

Reply:
I can remove the explicit installation of `python3.8` here, and I'll do that now, but `python3.8` will still be present as it's a dependency of `software-properties-common` (which depends on `python3`, which installs `python3.8` and makes it the default provider of the `python3` package).

Reply:
Good point on 3.8. we might wanna keep setuptools - some customers might be using it / depending on it?

Reply:
That sounds good, I'll leave setuptools in this PR.